tests/components/roborock/test_config_flow.py

```diff
@@ -261,6 +261,7 @@ async def test_reauth_flow(
     assert result["reason"] == "reauth_successful"
     assert mock_roborock_entry.unique_id == ROBOROCK_RRUID
     assert mock_roborock_entry.data["user_data"]["rriot"]["s"] == "new_password_hash"
+    await hass.async_block_till_done()
```

Comment:
Actually this is an odd place to put this. It should be under `async_configure`? 

But also there was just a change to the reload behavior and maybe that helps fix this problem?

Reply:
Can this line be removed if the other PR fixed it?

Still very good to bypass mqtt though so want to merge this.